deploy.sh

```diff
@@ -11,19 +11,16 @@ help() {
 EOH
 }
 deploy_generic() {
-    local environment="${1?No environment passed}"
```

Comment:
This is still needed, otherwise `$environment` would be empty (let me know if you want a longer explanation)